test/tests.js

```diff
@@ -1,79 +1,94 @@
 /* eslint-disable no-undef*/
-describe('OneTrust Forwarder', function() {
-    var server = new MockHttpServer(),
-        MockForwarder = function() {
-            var self = this;
-
-            this.initForwarderCalled = false;
-
-            this.apiKey = null;
-            this.OnConsentChangedCalled = false;
-            this.OnConsentChanged = function(fn) {
-                if (!self.OnConsentChangedCalled) {
-                    self.OnConsentChangedCalled = true;
-                } else {
-                    fn();
-                }
-            };
+var server = new MockHttpServer(),
+    MockForwarder = function() {
+        var self = this;
+
+        this.initForwarderCalled = false;
+
+        this.apiKey = null;
+        this.OnConsentChangedCalled = false;
+        this.OnConsentChanged = function(fn) {
+            if (!self.OnConsentChangedCalled) {
+                self.OnConsentChangedCalled = true;
+            } else {
+                fn();
+            }
         };
+    };
+
+function generateConsentGroupString(mapping) {
```

Comment:
I think `mockConsentGroupString` lets us know this is generating a test value rather than a real string.